R/annotate_driver_ssm.R

```diff
@@ -16,9 +16,11 @@
 #' @export
 #'
 #' @examples
+#' \dontrun{
```

Comment:
Even if this example is not executed, I think it makes sense to not refer to a no-longer-available dataset (Grande_maf). Can the example be updated to instead use the bundled maf (`GAMBLR.data::sample_data$grch37$maf`) and potentially even have the `dontrun{}` field removed?

Reply:
That is a great idea!
I updated it showing how would be done for GSC and non-GSC users.